graphix/visualization.py

```diff
@@ -344,7 +409,15 @@ def visualize_w_gflow(
             plt.savefig(filename)
         plt.show()
 
-    def visualize_wo_structure(self, angles=None, local_clifford=None, node_distance=(1, 1), save=False, filename=None):
+    def visualize_wo_structure(
```

Comment:
Is this currently showing entirely by random node placements? Is it possible to at least arrange nodes by their preparation or measurement order, perhaps (no need to be very tidy as there's no structure in this case

Reply:
I will address this in a separate issue.